definitions/infra-unix_host/definition.yml

```
@@ -0,0 +1,11 @@
+domain: INFRA
+type: UNIX_HOST
+synthesis:
+  name: hostname
+  identifier: hostname
```

```suggestion
  identifier: hostname
  conditions:
  - attribute: eventType
     prefix: unixMonitor:
```

cc @jfjoly this is correct right?